test/common/buffer/buffer_fuzz.cc

```diff
@@ -413,7 +413,7 @@ uint32_t bufferAction(Context& ctxt, char insert_value, uint32_t max_alloc, Buff
 
 void executeActions(const test::common::buffer::BufferFuzzTestCase& input, BufferList& buffers,
                     BufferList& linear_buffers, Context& ctxt) {
-  // Soft bound on the available memory for allocation to avoid OOMs and
+  // Soft bound on the available memory for allocation to avoid `OOMs` and
```

Comment:
Change to "OOM events" and it won't need to be quoted

Reply:
Done